### PR TITLE
Fix: quest chest system

### PR DESCRIPTION
**Pull request by UReddington**
This chest contains a small fix for the quest system, it removes the multiple messages on window and also removes a bug of invalid keys getting generated.

### CODE REVIEW

**Review by joao-blip — APPROVED** [review]

**Comment by adrianomengue** [comment]
@UReddington could you explain me how to reproduce the bug? So I can test your PR.

**Review by dudantas — COMMENTED** [review]
I did not understand. What was the correction?

File: data/scripts/actions/system/quest_reward_common.lua [hunk 1]
```diff
@@ -5,18 +5,19 @@
...
-Oswald
-Partos ***
-Quentin *
-Tark ***
-Harsky ***
-Stutch *
-Ferumbras *
-Frodo **
-Noodles ****]]
+			Hardek *
```
Comment (dudantas) [inline data/scripts/actions/system/quest_reward_common.lua]: You need to get back to how it was before. If you are not going to show this space that you gave, there in the book.

**Comment by UReddington** [comment]
> 
> 
> I did not understand. What was the correction?

When you open a chest with 2 a key it seems like you don't get it with the valid actionid (required to open the doors), but testing it recently i found that the fix isn't enough, so i wouldn't recommend to push this for now.

**Review by LukeAtMyNose — COMMENTED** [review]
In my test with the copper key 3801, the key now does not appear in the bag

image

**Comment by UReddington** [comment]
> 
> 
> In my test with the copper key 3801, the key now does not appear in the bag
> 
> image

this commit is incomplete, im going to close it until we have a better solution for this problem

**Pull request closed without merging**